README.md

```diff
@@ -3,22 +3,17 @@
 
 ## Intro
```

Comment:
Nit: Could you please call out this is an internal only bug bash, since this is a public repo that others could in theory discover.